### DISCUSSION TITLE
Error connecting to cnc

### DISCUSSION THREAD

**Discussion by TrainerLion** [winder/Universal-G-Code-Sender#2453] [Help]
Hello I have a cnc 3018 pro, it was working fine, but now it no longer connects me, I just closed the app and reopened it and it stopped working, but with GRBL controller it does work, I already tried deleting data, temporary files and downloading the app again.

The error is that there was a configuration issue in one or more of the following driver features:

Maximum Command Length
Long Decimal Truncated
Step-by-Step Mode
Remove Blank Spaces
Convert Arcs to Lines
Status Update
Status Update Rate

![image](https://github.com/winder/Universal-G-Code-Sender/assets/30944226/f9b81ff2-ed62-4b5e-b10c-c6d77100a528)

[messages.log](https://github.com/winder/Universal-G-Code-Sender/files/14173879/messages.log)

**Comment by breiler** [comment]
There is no errors in the log file. 

Can you post the file:
`C:\Users\wlope\.ugs\UniversalGcodeSender.json` and `C:\Users\wlope\.ugs\firmware_config\grbl.json`

**Reply by TrainerLion** [reply]
[UniversalGcodeSender.json](https://github.com/winder/Universal-G-Code-Sender/files/14179524/UniversalGcodeSender.json)

[grbl.json](https://github.com/winder/Universal-G-Code-Sender/files/14179529/grbl.json)

Thanks for the help

**Reply by breiler** [reply]
You have turned off the status polling which caused an error when connecting to a controller. A fix is on its way for this with better logging.

But you will need to have status polling activated in order to use GRBL:
![image](https://github.com/winder/Universal-G-Code-Sender/assets/8962024/e993fb2e-484d-477a-89e0-68d2ddf2a578)
